src/client/MetaRequest.cc

```diff
@@ -34,7 +34,11 @@ void MetaRequest::dump(Formatter *f) const
   f->dump_float("age", age.count());
   f->dump_int("mds", mds);
   f->dump_int("resend_mds", resend_mds);
-  f->dump_int("send_to_auth", send_to_auth);
+  f->dump_int("send_to_auth", send_to_auth || (head.op & CEPH_MDS_OP_WRITE)
+                                           || head.op == CEPH_MDS_OP_OPEN
+                                           || head.op == CEPH_MDS_OP_READDIR
+                                           || (head.op == CEPH_MDS_OP_GETATTR &&
+                                               (head.args.getattr.mask & CEPH_STAT_RSTAT)));
```

Comment:
NACK

Reply:
> NACK

@ukernel Any suggestions? Or leave it as is?

Reply:
Why not set `send_to_auth` appropriately in `choose_target_mds`?